site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
+* Paste the URL found below for the `Egeria-admin-services-operational` collection
+
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-operational.postman_collection.json
+    ```
+
+* Confirm the correct collection will be imported
+* Confirm the collection was imported successfully
+
+![Import Begin](postman-collection-import-begin.png)
+![Select import source](postman-collection-url.png)
+![Confirm Collection to import](postman-collection-importconf.png)
+![Confirm imported successfully](postman-collection-importcomplete.png)
+
+* Repeat the same procedure for the following Egeria collections:
+
+    `Egeria-admin-services-platform-configuration`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-platform-configuration.postman_collection.json
+    ```
+    `Egeria-admin-services-server-configuration`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-server-configuration.postman_collection.json
+    ```
+    `Egeria-asset-owner-omas`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/c31a2b19c3b42da1c2622dbe4ebed958f6a62cb5/open-metadata-implementation/access-services/asset-owner/Egeria-asset-owner-omas.postman_collection.json
+    ```
+    `Egeria-platform-services`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/platform-services/Egeria-platform-services.postman_collection.json
+    ```
+    `Egeria-repository-services-local-repository`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/repository-services/Egeria-repository-services-local-repository.postman_collection.json
+    ```
+    `Egeria-repository-services-metadata-highway`
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/e256aa1ca99240ac276b584f652cf02382220bf7/open-metadata-implementation/repository-services/Egeria-repository-services-metadata-highway.postman_collection.json
+    ```
+
+* Once all collections are imported, your workspace should appear as:
+
+![Show all imported collections](postman-show-imported-collections.png)
+
+We must now import the environment required for this dojo
+
+* Again, select 'Import' and select 'Link' as the source
+* Paste the sample environment URL found below
+
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-resources/open-metadata-samples/postman-rest-samples/Egeria.postman_environment.json
+    ```
+
+* Confirm the environment to be imported
+* Confirm the environment was imported successfully
+
+![No existing environments](postman-environments-empty.png)
```

Comment:
I suggest having text and twisties with these screenshots as well

Reply:
I've given more detailed instructions, and included the screenshot in a collapsible section.